includes/notifications/controllers/class.llms.notification.controller.quiz.passed.php

```diff
@@ -50,6 +57,62 @@ public function action_callback( $student_id = null, $quiz_id = null ) {
 
 	}
 
+	/**
+	 * Get an array of LifterLMS Admin Page settings to send test notifications
+	 * @param    string     $type  notification type [basic|email]
+	 * @return   array
+	 * @since    [version]
+	 * @version  [version]
+	 */
+	public function get_test_settings( $type ) {
+
+		if ( 'email' != $type ) {
+			return;
+		}
+
+		$query = new LLMS_Query_Quiz_Attempt( array(
+			'per_page'	=> 25,
+			'status'	=> 'pass',
+		) );
+
+		$options = array(
+		   '' => '',
+		);
+		$attempts = array();
+		$results = $query->results;
+
+		if ( $results ) {
+			foreach ( $results as $result ) {
+				$attempts[] = new LLMS_Quiz_Attempt( $result->id );
+			}
+		}
+
+		foreach ( $attempts as $attempt ) {
+			$quiz = llms_get_post( $attempt->get( 'quiz_id' ) );
+			$student = llms_get_student( $attempt->get( 'user_id' ) );
```

Comment:
@philwp `user_id` doesn't exist as one of the properties available on the `LLMS_Quiz_Attempt` model.

Since `llms_get_student()` falls back to `get_current_user_id()` when no user ID is supplied, this will result in your code APPEARING like it works when testing with a list of quizzes taken by your current (admin) user. However what's actually happening is that a WPDB error is encountered when calling `$attempt->get( 'user_id' )` and that's resulting an a `null` response from the getter. `null` is passed to`llms_get_student()` and then the current user shows up as the name for the list of quiz attempts.

Turn on `WP_DEBUG_DISPLAY` to see the WPDB error resulting from using `user_id` instead of `student_id`